source/reference/emphasis.txt

```diff
@@ -0,0 +1,71 @@
+.. _style-guide-emphasis:
+
+========
+Emphasis
+========
+
+.. default-domain:: mongodb
+
+Use **bold** or *italics* to add emphasis to text that you want to stand out on the
+page. Use ``monospace`` to emphasize code or key information.
+
+Emphasis Markup
+---------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 40 30
+
+   * - Format
+     - Markup
+     - Rendered
+
+   * - Bold
+     - ``**text**``
+     - **text**
+
+   * - Italic
+     - ``*text*``
+     - *text*
+
+   * - Monospace
+     - ````text````
+     - ``text``
+
+Emphasis Markup Rules
```

Comment:
Thought: I would find it helpful to know when we should use italics, and when we should use bold fonts. It's nice to now how, but I feel like we should also offer some guidance about WHY you should use one instead of the other. For instance, my team uses bold text for key words and phrases, and italics to emphasize relationships between ideas. I'm not sure what other teams do, but maybe you could survey them in #docs-writers.